ansible/playbooks/molecule/teleop_empty_server_docker/tests/test_teleop_server.py

```diff
@@ -33,18 +33,24 @@ def test_docker_container_exists(host):
 
 def test_correct_docker_image(host):
     client = docker.from_env()
-    image = str(client.containers.get('dexterous_hand_real_hw').image)
+    image = str(client.containers.get('teleop').image)
     assert image == "<Image: 'shadowrobot/dexterous-hand:kinetic-release'>"
 
 
 def test_sr_config_exists_in_docker(host):
     client = docker.from_env()
-    container = client.containers.get('dexterous_hand_real_hw')
+    container = client.containers.get('teleop')
     bits, stat = container.get_archive(
         '/home/user/projects/shadow_robot/base/src/sr_config')
     assert stat['size'] > 0
 
 
 def test_icons_in_docker(host):
-    f = host.file('/home/Desktop/Teleop_Launcher.desktop')
-    assert f.exists
+    user = host.user().name
+    icon_list = []
+    desktop_path = '/home/'+str(user)+'/Desktop/'
```

Comment:
please add spaces before and after plus sign

Reply:
Fixed in the last commit